service/nodeconfig/v1/resource/node/create.go

```diff
@@ -83,35 +82,26 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 
 		for _, p := range podList.Items {
 			if p.GetNamespace() == "kube-system" {
-				fmt.Printf("systemPods: %#v\n", p.GetName())
 				systemPods = append(systemPods, p)
 			} else {
-				fmt.Printf("customPods: %#v\n", p.GetName())
 				customPods = append(customPods, p)
 			}
 		}
 	}
 
-	{
-		fmt.Printf("\n")
-		fmt.Printf("customPods\n")
-		for _, p := range customPods {
-			fmt.Printf("%#v\n", p)
+	for _, p := range customPods {
+		err := k8sClient.CoreV1().Pods(p.GetNamespace()).Delete(p.GetName(), &apismetav1.DeleteOptions{})
+		if err != nil {
+			return microerror.Mask(err)
 		}
-		fmt.Printf("customPods\n")
-		fmt.Printf("\n")
+	}
```

Comment:
We delete the custom pods first, if any. I think we will have to make real world experiences with eventual buffers to let all workloads go down gracefully. I do not want to over engineer this now and solve problems that do not exist. Lets iterate.